sdk/containerservice/Azure.ResourceManager.ContainerService/src/autorest.md

```diff
@@ -50,5 +80,57 @@ rename-rules:
   SSO: Sso
   URI: Uri
   Etag: ETag|etag
+  SSD: Ssd
+  GPU: Gpu
+  SKU: Sku
+  AAD: Aad
+  TrustedCa: TrustedCA
+  CBLMariner: CblMariner
+  API: Api
+  OCI: Oci
+  CSI: Csi
+  MIG1G: Mig1G
+  MIG2G: Mig2G
+  MIG3G: Mig3G
+  MIG4G: Mig4G
+  MIG7G: Mig7G
+  Tcpkeepalive: TcpKeepalive
 
+override-operation-name:
+  ResolvePrivateLinkServiceId_POST: ResolvePrivateLinkServiceId
+  AgentPools_GetAvailableAgentPoolVersions: GetAvailableAgentPoolVersions
+
+prepend-rp-prefix:
+  - TimeSpan
+  - TimeInWeek
+  - WeekDay
+
+directive:
+  - from: managedClusters.json
+    where: $.definitions.AgentPoolAvailableVersionsProperties.properties.agentPoolVersions.items
+    transform: >
+      $['x-ms-client-name'] = 'AgentPoolAvailableVersion';
```

Comment:
not move to `rename-mapping`?

Reply:
It's an anonymous class with no model defined in swagger, not supported by `rename-mapping`.